Keyword Analysis.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[3]:
+
+
+import pandas as pd
+import numpy as np
+from sklearn.feature_extraction.text import CountVectorizer
+import nltk
+from nltk.corpus import stopwords
+
+# reading csv files into pandas dataframes
+hist_BK = pd.read_csv("historical_search_BK.csv")
+hist_SI = pd.read_csv("historical_search_SI.csv")
+
+# countvectorizing BK tweets
+
+vectorizer = CountVectorizer(ngram_range=(1,3))
+
+X = vectorizer.fit_transform(hist_BK['tweet'])
+
+col_names = vectorizer.get_feature_names()
+
+col_names.insert(0,'user_id')
+
+user_ids = pd.DataFrame(hist_BK['user_id'],columns=['user_id'])
+
+tweets = pd.DataFrame(X.toarray(),columns=vectorizer.get_feature_names())
+
+# build df with BK user_ids and countvectorized tweets
+df_BK = pd.concat([user_ids,tweets],axis=1)
+
+# import list of general stopwords
+stopwords = stopwords.words()
+
+# create list of stopwords found in the BK tweets
+stopwords_BK = list(set(df_BK.columns) & set(stopwords))
+
+# remove stopwords from countvectorized BK tweets
+df_BK = df_BK.drop(stopwords_BK,axis=1)
+
+
+# In[4]:
+
+
+df_BK.iloc[:,1:].sum(axis=0).sort_values(ascending=False)[:50]
```

Comment:
seems pretty nice barclays was at the top, since its pretty specific. was there a protest there??

Reply:
yah so the location i set for brooklyn tweets was centered around grand army plaza and included barclays center; i chose this location because that was where the largest protests in brooklyn were happening so i thought it might have more relevant tweet activity